### ISSUE TITLE
Checkmark icon is not shown on standard select boxes

### ISSUE THREAD

**Issue by ahamana** [snapappointments/bootstrap-select#2126]
After upgrading to v1.13.3, the checkmark icon is not shown on standard select boxes with the show-tick class.
You can see this issue at the "Checkmark on selected option" section on the attached archive file.
- v1.13.2.html
  Checkmark icon is shown.
- v1.13.3.html
  Checkmark icon is not shown.

[reproduce.zip](https://github.com/snapappointments/bootstrap-select/files/2521238/reproduce.zip)

**Comment by klaymen** [comment]
The issue is still valid with v1.13.8.
This might be related: https://github.com/snapappointments/bootstrap-select/issues/1425

In case you set the selectpicker and show-tick classes:
```
<select class="selectpicker show-tick">
  <option>Mustard</option>
  <option>Ketchup</option>
  <option>Relish</option>
</select>
```
 and, you initialize from js, with the proper setting:

```
$( document ).ready(function() {
  $('select').selectpicker({
    showTick: true,
  });
});
```
it is working. However, this is rather a workaround than a solution.

**Comment by caseyjhol** [comment]
Released in [v1.13.9](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.9)!

**Comment by CapDev** [comment]
After upgrading to 1.13.9 the checkmark icon is no longer showing for me on standard select boxes when using  $.fn.selectpicker.Constructor.DEFAULTS.showTick = true;  and not including the show-tick class with the select element.

In 1.13.8 and earlier the checkmark icon will display without having to add the show-tick class as it added by bootstrap-select when you change the showTick option to true as a constructor default or as an option in the javascript .selectpicker() call.

Is it required now to add the show-tick class to selects when needed instead of or along with using the showTick option?

Thanks for the great plugin.

**Comment by caseyjhol** [comment]
Released in [v1.13.10](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.10)!
